package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
The source is not ES2015 – it's whatever is valid Babel stage 0.

Moreover the target for `jsnext:main` isn't ES2015 anyway – it's ES5 with ES modules (rather than CJS modules).